content/en/apps/reference/app-settings/_index.md

```diff
@@ -44,11 +44,63 @@ The following settings do not need to be specified. They should only be defined
 
 Workflows involving SMS are configured by defining [schedules]({{< relref "schedules" >}}), [registrations]({{< relref "registrations" >}}), [patient reports]({{< relref "patient_reports" >}}), and [case reports]({{< relref "accept_case_reports" >}}). Schedules of automated messages can be sent from the server at specificied times in the future, and reports can be associated to contacts.Forms can also be configured to clear the schedule, or silence it for a period of time.
 
+## SMS recipient resolution
+
+An outgoing SMS message configuration has the following fields:
+
+|property|description|required|
+|-------|---------|----------|
+|`translation_key`|The translation key of the message to send out. Available in 2.15+.|yes|
+|`messages`| (**deprecated**) Array of message objects, each with `content` and `locale` properties. From 2.15 on use `translation_key` instead.|no|
+|`recipient`| Recipient of the message.|no|
+
+### `recipient` values and resolutions:
+
+|value|resolves to|
+|-----|-----------|
+|*empty*| submitter |
+|`reporting_unit`| submitter | 
+|`parent`| primary contact of the patient's/submitter's place's parent (`patient.parent.parent.contact`) | 
+|`grandparent`| primary contact of the patient's/submitter's place's grandparent (`patient.parent.parent.parent.contact`) |
+|`clinic`| primary contact of the `clinic` in the patient's/submitter's lineage | 
+|`health_center`| primary contact of the `health_center` in the patient's/submitter's lineage | 
+|`district`| primary contact of the `district_hospital` in the patient's/submitter's lineage |
+|`ancestor:<contact_type>`| primary contact of the place of the requested type in the patient's/submitter's lineage |
+|`link:<tag>`| Linked doc that has requested `tag` in the patient's / submitter's lineage (direct mapping, not to primary contact). *As of 3.10.x*| 
+|`link:<contact_type>`| primary contact of the place of the requested `contact_type` in the patient's/submitter's lineage. *As of 3.10.x* | 
+| *custom object path* | a direct object path in the [message context object](#message-context) eg: `patient.parent.contact.other_phone` | 
+| *valid phone number* | requested phone number |
+
+{{% alert title="Note" %}} 
+- if `recipient` resolution does not yield a phone number, it will default to submitter's phone number
+- if there is no submitter phone number available, the actual `recipient` property value will be used
+- when mapping a contact phone number, `patient` lineage and linked_docs take precedence over `submitter` lineage and linked_docs. 
```

Comment:
I am not so familiar with `linked_docs` yet... If it is a field it should be wrapped with back ticks, and otherwise it may be better to spell it out as two words.

Reply:
Done. `linked_docs` documentation update is actually part of this PR, sprinkled in the hydration and has a full section in `db-schema`.